### ISSUE TITLE
Merge end-of-multiplayer box with lobby

### ISSUE THREAD

**Issue by SimonN** [SimonN/LixD#242]
Lix 0.9

Observers have to cancel the end-of-mutliplayer box. Observers should not have to click anything at all. It's nice to save the replay, but it's even more important to have no interaction.

Short-term solution is to print to console.

Long-term solution is what? Create extra space to right of console, print table to that space?

**Comment by wrathix** [comment]
Or observers have a behind the scenes countdown that sends them back to the lobby in 5 seconds after game ends?

**Comment by SimonN** [comment]
The box is gone, but I still have to draw the ScoreBoard in the lobby. Thus, this bug is still open.

**Comment by SimonN** [comment]
> Or observers have a behind the scenes countdown that sends them back to the lobby in 5 seconds after game ends?

Right, anything involving extra screens or boxes needs special care for observers. The goal is still to allow observers input-free observing of several successive games.

I'm considering an entire screen with end-of-game statistics, including who saved which lixes. ("Ah, player X donated the most to player Y and that's why player Y won, player Z wasn't involved in the donation after all.") And possibly list how many died to traps/water/oblivion/..., then the frog can compete with the players. :-)
